### PR TITLE
Some improvements to the Repostitory

### PR DESCRIPTION
**Pull request by qtGenn**
I have overhauled the README.md file, making it more organized and less cluttered. I have also improved .gitattributes file, which changes you can check yourself (I can revert it if needed). ~~I have also decided to move folders and files to a "content" folder in order to reduce clutter on the main page, which might not be a welcomed change but I felt that it would be a great improvement.~~

### CODE REVIEW

**Comment by KeinNiemand** [comment]
I'll look at this in more detail and If I like the changes I'll merge. 
Why does it say it removes the mimalloc submodules, is that a leftover from the move to a content folder that you reverted? The mimalloc submodules should not be removed. I think the revert commit didn't revert the removal of the submodule.
Sorry for the late response I don't look at my GitHub that frequently, but thanks for contributing any imp.

File: Lib/Submodules/mimalloc
Comment (KeinNiemand) [inline Lib/Submodules/mimalloc:1]: Why is the submodule removed?

Comment (qtGenn) [reply Lib/Submodules/mimalloc:1]: I am not sure; I have tried to add it back after reverting the source code movement, but it for some reason wouldn't add it back. Perhaps it has something to do with VSCodium that I am using.

**Comment by KeinNiemand** [comment]
When you reverted those content changes did you manually undo them or did you revert the commit with git revert? git revert should undo it perfectly.
Maybe try and make a new branch (starting from my master) and then use git cherry-pick to get all the excluding everything involving the content folder changes (and your attempts of reverting the changes
Submodules are not defined by the file they are part of the git repo itself so you need to add/remove/modify them with git submodule commands.

**Comment by KeinNiemand** [comment]
Looking at your actual readme changes, your added disclaimer is wrong since LargePageInjectorMods does not directly require Admin Privileges.

The LockPagesInMemory Privilege required for Large Pages to work does not grant any sort of Admin privileges, it just allows locking pages in memory(which stops them from ever getting paged out)/using large pages. 
The reason this is disabled by default is presumably (I don't actually know why and am just guessing) the fragmentation issues Large Pages can create and/or that an application could presumably crash your system by consuming all your ram and preventing it's ram from being moved out. 
The Microsoft documentation you linked also doesn't mention any security consideration besides something talking up all your ram and making the system unresponsive/crashing.

The only part that actually requires admin is giving your user that privilege, but that's just needing admin to change a setting in Windows, not LargePageInjectorMods ever having access to/needing admin itself.

Please update the disclaimers to be more acurate.

**Pull request closed without merging**